test/verify/check-system-services

```diff
@@ -612,9 +612,22 @@ WantedBy=default.target
 
             # system timers are not allowed to be deleted
             self.goto_service("cockpit-system.timer")
-            b.click(".service-top-panel .pf-v5-c-dropdown button")
-            b.wait_in_text(".service-top-panel .pf-v5-c-dropdown__menu", "Disallow running")
-            b.wait_not_in_text(".service-top-panel .pf-v5-c-dropdown__menu", "Delete")
+            # Check our custom kebab menu in RTL layout separately as the tests switch to RTL layout by editing
+            # html directly while in a real life scenario we would reload the page. This circumvents the PF dropdown
+            # from rendering in the wrong place when making a pixel test.
+            b.set_layout("rtl")
+            b.click(".service-top-panel button.pf-v5-c-menu-toggle")
+            b.wait_in_text(".service-top-panel .pf-v5-c-menu__list", "Disallow running")
+            b.assert_pixels_in_current_layout("#service-details-unit", "services-kebabdropdown-rtl")
```

Comment:
This produced [this reference image](https://github.com/cockpit-project/pixel-test-reference/blob/0cb93097308d61b17208f033c232c9ce54cea8de/TestServices-testTimer-services-kebabdropdown-rtl-rtl-pixels.png). That is LTR though? Perhaps that needs some waiting, i.e. set_layout() isn't synchronous?

Also, the name is "*-rtl-rtl", supposedly assert_pixels() already appends one -rtl suffix.

Reply:
Arggh this indeed wrong...

https://cockpit-logs.us-east-1.linodeobjects.com/pull-20117-20240304-103727-90c51cd7-fedora-39-other/TestServices-testTimer-services-kebabdropdown-rtl-pixels.png

Is correct

Reply:
Well guess what? `set_layout` does not handle RTL :)

Reply:
[Perfect!](https://github.com/cockpit-project/pixel-test-reference/blob/d5a0de20d062608d3d7b83c5f208895fe4be73e1/TestServices-testTimer-services-kebabdropdown-rtl-rtl-pixels.png)

Thanks for figuring this out!